.github/workflows/build.yml

```diff
@@ -545,20 +545,24 @@ jobs:
     - name: Tests
       run: xvfb-run make test
 
-  build-tsan:
-    name: >-
-      Thread sanitizer
-      ${{ fromJSON(matrix.free-threading) && '(free-threading)' || '' }}
+  build-san:
+    name: >-  # ${{ '' } is a hack to nest jobs under the same sidebar category
+      Sanitizers${{ '' }}
     needs: build-context
     if: needs.build-context.outputs.run-tests == 'true'
     strategy:
       fail-fast: false
       matrix:
+        check-name:
+        - Thread
         free-threading:
         - false
         - true
-    uses: ./.github/workflows/reusable-tsan.yml
```

Comment:
Why not remove the `reusable-tsan.yml` file?

Reply:
I'm confused. The file doesn't exist in 3.14 & main. Is this about being close to 3.12?

Reply:
An unused file doesn't sound good for maintainability.

Reply:
It was renamed from `tsan` to just `san` in #136820. This patch backports that renaming bit.

Reply:
Makes sense now -- you just forgot to commit the removal part of the rename, right?

Reply:
Oh, I didn't realize that 🤦
Thanks for noticing!